cloud_testing/kotlin_poc/src/main/kotlin/ftl/gc/GcAndroidTestMatrix.kt

```diff
@@ -70,7 +70,7 @@ object GcAndroidTestMatrix {
         testMatrix.testSpecification = TestSpecification()
                 .setAndroidInstrumentationTest(androidInstrumentation)
                 .setDisablePerformanceMetrics(config.disablePerformanceMetrics)
-                .setDisableVideoRecording(config.disableVideoRecording)
+                .setDisableVideoRecording(config.recordVideo)
```

Comment:
I think this has the same logic error as `disablePerformanceMetrics`

Reply:
Ah, now I get why they were named them that way, thought it was kind of counterintuitive, haha